watertap/examples/flowsheets/nf_dspmde/tests/test_nf.py

```diff
@@ -20,12 +20,12 @@
 def test_main():
     m = main()
     test_dict = {
-        "lcow": [m.fs.costing.LCOW, 0.16811587158493219],
-        "pressure": [m.fs.NF.pump.outlet.pressure[0] / 1e5, 6.56],
-        "area": [m.fs.NF.nfUnit.area, 285.6900547389303],
+        "lcow": [m.fs.costing.LCOW, 0.15058960529129017],
+        "pressure": [m.fs.NF.pump.outlet.pressure[0] / 1e5, 8.13],
+        "area": [m.fs.NF.nfUnit.area, 423.8956418211484],
         "recovery": [
             m.fs.NF.nfUnit.recovery_vol_phase[0.0, "Liq"] * 100,
-            73.47934090302432,
+            94.99999441324391,
         ],
```

Comment:
This test has a larger difference in results. @bknueven @adam-a-a

Reply:
@Zhuoran29 can you double check the value of the `total_operating_cost` here, both before and after the change?

Reply:
@bknueven Sure, before the change `total_operating_cost` = 2369.18, and after the change `total_operating_cost` = 2429.43.

Reply:
I was able to do this myself:
current main: `total_operating_cost: 2369.1822124750583`
With this change: `total_operating_cost: 2429.430766595918`

It's actually a bit higher, funnily enough ...

Reply:
This change also brings the converged values without bypass much closer to those with by pass:

No Bypass, `main`:
```
total_operating_cost: 2369.182212475059
Optimal cost 0.16810666231247273
Optimal NF pressure (Bar) 6.560034522531212
Optimal area (m2) 285.7101510399154
Optimal nf recovery (%) 73.48423398501477
Feed hardness (mg/L as CaCO3) 1016.1545749479138
Product hardness (mg/L as CaCO3) 199.9999928713816
Disposal hardness (mg/L as CaCO3) 3277.997368528139
```

No Bypass, with this change:
```
total_operating_cost: 2429.430766595918
Optimal cost 0.15058960529129017
Optimal NF pressure (Bar) 8.12906860029318
Optimal area (m2) 423.89564182114776
Optimal nf recovery (%) 94.99999441324391
Feed hardness (mg/L as CaCO3) 1016.1545749479138
Product hardness (mg/L as CaCO3) 199.9995335609493
Disposal hardness (mg/L as CaCO3) 16523.08212268398
```

With Bypass with / without this change:
```
total_operating_cost: 2217.7019485486526
Optimal cost 0.1376886456374725
Optimal NF pressure (Bar) 6.702781481688757
Optimal area (m2) 419.7559334394971
Optimal nf recovery (%) 94.99999897882932
bypass (%) 10.592027708476932
Feed hardness (mg/L as CaCO3) 1016.1240639142685
Product hardness (mg/L as CaCO3) 199.9999851467023
Disposal hardness (mg/L as CaCO3) 18456.17762054043
```